packages/lime-system/files/usr/lib/lua/lime/config.lua

```diff
@@ -71,8 +75,8 @@ function config.get_all(sectionname)
 	return nil
 end
 
-function config.get_bool(sectionname, option)
-	local val = config.get(sectionname, option)
+function config.get_bool(sectionname, option, default)
+	local val = config.get(sectionname, option, default)
```

Comment:
Please double check if config.autogenerable(section_name) keep working with this new implementation

Reply:
Why should it not work? The default parameter is the last parameter so it can be just "nil".